actions/codelist/analysis/weekly_summary.py

```diff
@@ -0,0 +1,31 @@
+# from study_utils import create_top_5_code_table
```

Comment:
This seems to be a new module that isn't used anywhere. What's its purpose?

Reply:
I think I've made a snafu and based this work on an older version of the report code :facepalm:

Reply:
removed. It's in the interactive code atm, but its not used